### ISSUE TITLE
Add informative text cross-walking OARec concepts with STAC concepts.

### ISSUE THREAD

**Issue by pvretano** [opengeospatial/ogcapi-records#306]
It was suggested, at the Sept 2023 members' meeting, to add information text to the specification cross walking OGC API Records concepts with STAC concepts.

**Comment by m-mohr** [comment]
So pretty much this?
https://github.com/stac-utils/stac-crosswalks/tree/master/ogcapi-records

**Comment by m-mohr** [comment]
By the way, we also discussed the Records / STAC alignment during our STAC sprint.

We will align the license field with Records.
The required fields that record requires, but STAC doesn't require we can't require in a 1.1 as it would be breaking. Also, for some fields it doesn't make sense in STAC (e.g. title in an Item - are you sure you want to require that?)

We discussed having a STAC extension that requires those fields and adds things to STAC that are missing from Records Core. If clients then find this extension in the stac_extensions array, they can assume that the STAC is also a valid OGC API - Records extension.
